tck/src/main/java/org/eclipse/microprofile/rest/client/tck/InvokeWithRegisteredProvidersTest.java

```diff
@@ -97,7 +97,7 @@ public void testInvokesPutOperationWithAnnotatedProviders() throws Exception {
             .baseUrl(getServerURL())
             .build(InterfaceWithProvidersDefined.class);
 
-        Response response = api.executePut(id, inputBody);
+        Response response = api.executePut(expectedId, inputBody);
```

Comment:
This test checks that the providers are invoked, not the server behavior.

Reply:
See my comment above